testing/file_test/test_file.h

```diff
@@ -58,9 +58,9 @@ struct TestFile {
   // Files in the test, generated by content and splits.
   llvm::SmallVector<Split> file_splits;
 
-  // // Paths to files to be included in the test. They are added to `test_args`
+  // Paths to files to be included in the test. They are added to `test_args`
```

Comment:
These are no longer "Paths", and are not added to `test_args`

Reply:
Rewrote:

```
  // Files pulled in by `INCLUDE-FILE`. They're combined with `file_splits` for
  // execution.
```